host-configs/TOTAL/pangea3-gcc8.4.1-openmpi-4.1.2.cmake

```diff
@@ -77,7 +77,6 @@ set(ENABLE_MATHPRESSO OFF CACHE BOOL "")
 # Silo configure script doesn't recognize systype
 set(SILO_BUILD_TYPE powerpc64-unknown-linux-gnu CACHE STRING "")
 
-set(GEOS_BUILD_SHARED_LIBS OFF CACHE BOOL "")
```

Comment:
It is not possible to build Geos as a shared library on powerpc architecture (error at link edition).

As by default the `GEOS_BUILD_SHARED_LIBS` variable is enabled, we have to keep this line for the P3 host-config (at least until we fix the linke error, which is not a priority for us for now).